app/lib/esQueryBuilder.js

```diff
@@ -5,13 +5,73 @@ function build(searchTerm) {
     body: {
       size: 30,
       query: {
-        multi_match: {
-          query: searchTerm,
-          fields: ['name^2', 'address.addressLines', 'doctors'],
-          type: 'cross_fields',
-          operator: 'and'
+        bool: {
+          should: [
+            {
+              match_phrase: {
+                name: {
+                  query: searchTerm,
+                  boost: 2,
+                  slop: 1
+                }
+              }
+            },
+            {
+              match_phrase: {
+                alternativeName: {
+                  query: searchTerm,
+                  boost: 2,
+                  slop: 1
+                }
+              }
+            },
+            {
+              common: {
+                name: {
+                  query: searchTerm,
+                  cutoff_frequency: 0.0001
+                }
+              }
+            },
+            {
+              common: {
+                alternativeName: {
+                  query: searchTerm,
+                  cutoff_frequency: 0.0001
+                }
+              }
+            },
+            {
+              nested: {
+                path: 'doctors',
+                query: {
+                  bool: {
+                    should: [
+                      {
+                        match_phrase: {
+                          'doctors.name': {
+                            query: searchTerm,
+                            boost: 2,
+                            slop: 1
+                          }
+                        }
+                      },
+                      {
+                        match: {
+                          'doctors.name': {
+                            query: searchTerm,
+                            cutoff_frequency: 0.0001
```

Comment:
My understanding of how this [cutoff_frequency](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query.html#query-dsl-match-query-cutoff) works value is that words appearing more than once in every 10,000 documents will be classified as common. Is that going to lead to common surnames such as `jones`, `smith`, `ali`, `ahmed`, etc. being considered low frequency? Not sure if that will have any impact on results?
I'm getting counts of between ~150 and 230 using this query:
` jq '.[].doctors' data/input/gp-data-merged.json | grep -i ' <name>"' | wc -l` when run in `profiles-db-elastic`.